src/components/Form.js

```diff
@@ -95,7 +99,7 @@ class Form extends React.Component {
 
         this.state = {
             errors: {},
-            inputValues: {},
+            inputValues: this.props.draftValues,
```

Comment:
NAB, but if you want we can remove the `this` since `props` is also in scope for the `constructor()`
```suggestion
            inputValues: props.draftValues,
```